runtime/src/lib.rs

```diff
@@ -176,22 +176,22 @@ where
 		E: Ext<T = Runtime>,
 		<E::T as SysConfig>::AccountId: UncheckedFrom<<E::T as SysConfig>::Hash> + AsRef<[u8]>,
 	{
+		let mut env = env.buf_in_buf_out();
+		let contracts_overhead = <Runtime as pallet_contracts::Config>::Schedule::get()
+			.host_fn_weights
+			.debug_message;
```

Comment:
:+1:  I would also add a comment with some explanation

Reply:
I took a swing at it